src/App.tsx

```diff
@@ -1,109 +1,31 @@
-import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { User } from './types/User';
+import { UserComment } from './types/UserComment';
+
+const posts = postsFromServer.map(post => {
+  const user = usersFromServer
+    .find(item => item.id === post.userId) as User;
+  const comments = commentsFromServer
+    .filter(comment => comment.postId === post.id) as UserComment[];
```

Comment:
when you say `as`, you effectively disable typescript. there is time and place for that, but here is not a good example